packages/components/filters/src/filter-menu/trigger-button/trigger-button.styles.ts

```diff
@@ -15,7 +16,7 @@ export const triggerWrapper = css`
   box-shadow: 0 0 0 ${designTokens.borderWidth1} ${designTokens.colorNeutral85};
   cursor: pointer;
   height: ${designTokens.spacing50};
-  max-width: 100%;
+  max-width: ${Constraints.getMaxPropTokenValue(13)};
```

Comment:
What was the reason to change the value? If it's not 100% (="consume not more than the available horizontal space") it may ignore potential boundaries set by a parent component.

e.g. I limit the width available to the FilterMenu:

```
<div style={{ width: '256px'}}>
    <FilterMenu {...} />
</div>
```

This is what happens:

![image](https://github.com/user-attachments/assets/60b3df6f-d313-4b25-a433-bd317dd0b87c)

Reply:
its because the [figma specifies](https://www.figma.com/design/b2S3GGpScVDuf15VAGEqeL/Filter-pattern?node-id=2339-7258&m=dev) that the badge should display if the button is wider than `642px`.

This is a good point though. I think adding a width of 100% satisfies both conditions?

```suggestion
  width: 100%;
  max-width: ${Constraints.getMaxPropTokenValue(13)};
```
which results in:
![Screenshot 2024-10-15 at 11 24 40 AM](https://github.com/user-attachments/assets/20e37fba-64e6-422d-85db-1e07705c4400)

Reply:
this is because of the requirement that the badge container be displayed when the component gets to be [larger than 642px](https://www.figma.com/design/b2S3GGpScVDuf15VAGEqeL/Filter-pattern?node-id=2339-7258&m=dev).

I think this works to satisfy both requirements:
```suggestion
  max-width: min(100%, ${Constraints.getMaxPropTokenValue(13)});
```
![Screenshot 2024-10-15 at 12 52 54 PM](https://github.com/user-attachments/assets/411e11fc-cbed-4f79-bdc3-3c65a0292122)